README/INSTALL

```diff
@@ -150,201 +118,18 @@ Using the fixed location mode (section 3.2), the default 'CMAKE_INSTALL_PREFIX'
 path is '/usr/local', which will result in the ROOT files to be
 installed in '/usr/local/bin', `/usr/local/lib', etc. You can specify
 an installation prefix other than `/usr/local' by giving `cmake'
```

Comment:
Shall we suggest `/opt/root-6.22/` "as that makes it easy to uninstall ROOT and to keep multiple versions"?

Reply:
yes!